backend/config/config.go

```diff
@@ -191,6 +195,8 @@ func NewDefaultAppConfig() AppConfig {
 			LitecoinP2WPKHActive:     true,
 			EthereumActive:           true,
 
+			SplitAccounts: false,
```

Comment:
Nicely chosen. This should work for both a new app where no config exists yet and already installed apps.

Reply:
Actually, ignore this. It's only in my head.